README.md

```diff
@@ -76,16 +76,27 @@ They have a short header and then the page content in the form:
 
 /---
 layout: default
-title: Getting Started with OMERO.insight Version 5.0.3
+title: Getting Started with OMERO.insight
 menu-id: l1-quickstart
 pdfs:
     - getting-started-5.pdf
 description: Metadata header for each page.
 version:
     - yes
+redirect_from:
+  - /old-page.html
 /---
 <page content>
 
+These are used as follows:
```

Comment:
Again, line breaks not rendered, you probably want to format this as a bullet point list